include/trick/SysThread.hh

```diff
@@ -0,0 +1,54 @@
+/*
+    PURPOSE:
+        (Trick Sys Threads implementation)
+*/
+
+#ifndef SYSTHREAD_HH
+#define SYSTHREAD_HH
+
+#include <stdio.h>
+#include <pthread.h>
+#include <iostream>
+#include <string>
+#include <vector>
+#if __linux
+#include <sys/types.h>
+#endif
+#include <unistd.h>
+#include <sched.h>
+#include "trick/ThreadBase.hh"
+
+
+namespace Trick {
+
+    /**
+     * The purpose of this class is to ensure safe shutdown for Trick system threads, since user threads are handled separately in
+     * the Trick::Threads and Executive classes.
+     *
+     * @author Jackie Deans
+     *
+     *      
+     **/
+    class SysThread : public Trick::ThreadBase {
```

Comment:
Could you add a comment that references issue 1445, that specifies the motivation for creating this class?